launchlib/launcher.go

```diff
@@ -95,13 +95,18 @@ func compileCmdFromConfig(
 			staticConfig.JavaConfig.Classpath))
 		_, _ = fmt.Fprintf(logger, "Classpath: %s\n", classpath)
 
+		var combinedJvmOpts []string
+		combinedJvmOpts = append(combinedJvmOpts, staticConfig.JavaConfig.JvmOpts...)
+		combinedJvmOpts = append(combinedJvmOpts, customConfig.JvmOpts...)
```

Comment:
note: This can be 1 line as well: 
```go
combinedJvmOpts := append(staticConfig.JavaConfig.JvmOpts, customConfig.JvmOpts...)
```

Reply:
Does this not modify staticConfig.JavaConfig.JvmOpts backing array/slice?

Reply:
Technically it can modify the backing array, but given both are slices it should be fine. It all depends on the size of the original slice `jvmOpts`. Let's just keep it the way you have it written to avoid any ambiguity in the backing array.

Reply:
yeah, I think it's rather not modify the originals, keep it nice and read-only.